### PR TITLE
fix wrong processing unsupported urls on server side (fix #474)

### LINKED ISSUE #474: A lot of 404 errors in HHM
Seems like the cause is #466

### PR DESCRIPTION
**Pull request by miherlosev**
/cc @churkin @inikulin

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

**Comment by churkin** [comment]
gltm

File: src/processing/resources/index.js [hunk 1]
```diff
@@ -9,6 +9,9 @@ import { ensureTrailingSlash } from '../../utils/url';
 
 function getResourceUrlReplacer (ctx) {
     return function (resourceUrl, resourceType, charsetAttrValue, baseUrl) {
+        if (!urlUtil.isSupportedProtocol(resourceUrl))
```
Comment (inikulin) [inline src/processing/resources/index.js]: So what was the cause of the problems and how what fixes it? Also missing regression tests

Comment (miherlosev) [inline src/processing/resources/index.js]: ### Algorithm
If we support the protocol then we should proxy url. If not then we shoul kept as is.
see client implementation - https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/url.js#L10

On server side exception was raised if protocol is not supported.
So in https://github.com/DevExpress/testcafe-hammerhead/pull/466 we remove error generation for non-supported protocol.
After then we became on server side calculate proxy url for all protocols.

### Regression test
I extend the existing test `Should process stylesheets` (see https://github.com/DevExpress/testcafe-hammerhead/blob/master/test/server/proxy-test.js#L491).
It use two files that was changed:
test/server/data/stylesheet/expected.css
test/server/data/stylesheet/src.css

Comment (inikulin) [inline src/processing/resources/index.js]: But why we don't throw like we did before if protocol is not supported?

**Comment by inikulin** [comment]
lgtm

**Pull request merged**